app/assets/javascripts/app/lib/app_post/searchable_select.coffee

```diff
@@ -226,6 +225,7 @@ class App.SearchableSelect extends Spine.Controller
     @input.trigger('change')
     @shadowInput.val event.currentTarget.getAttribute('data-value')
     @shadowInput.trigger('change')
+    $(".js-shadow-ids[value=#{event.currentTarget.getAttribute('data-value')}]").closest('div').remove()
```

Comment:
What is this line for?

If we keep it:
It's not really expressive. We should add a comment for it and extend the tests to cover the functionality.